### DISCUSSION TITLE
Clear history of a habit

### DISCUSSION THREAD

**Discussion by t214c** [iSoron/uhabits#1753] [Feature Requests]
A way to delete all previously-entered data points, but keep the habit itself.

**Comment by exanrauzer** [comment]
What's the point? Making new habit leads to the same with almost zero effort.

**Reply by t214c** [reply]
I periodically move the data to a file and want to know which ones have already been moved (move all > clear > anything present is new and is yet to be moved).

**Reply by exanrauzer** [reply]
I would consider it as a rare kind of approach and app usage and I think It is User's side workaround to cover User's need you described. But let me know I am wrong.

**Comment by DeveloperJF** [comment]
Yo acabo de hacer una solicitud sobre este mismo punto y no me había dado cuenta de este; pero el comentario es el siguiente:

Mi caso es que quiero que el histórico sea reiniciado porque yo hace mucho que no había utilizado nuevamente la App, por ende me gustaría reiniciarlo para estar más claro, en el tema de la estadística, porque empezando hoy hacía futuro me va a tomar desde el año 2023 y no desde el 2017 descargué la App y creé el hábito, por ende esto me va a generar unos valores distintos y para que no me afecten estos me gustaría poder reiniciarlo, sin perder las características con las que este fue creado; porque la idea no es eliminar el hábito, si no reiniciar sus valores internos de rachas, frecuencia, calendario, historial, puntuación, resumen; es volver a cero estos valores.

**Comment by alr86** [comment]
I also searched for "Clear History" in the GitHub discussions in order to add myself if anyone hasn't requested. I have a lot of habits, so I prefer to be such an option.
